cli/src/commands/read.ts

```diff
@@ -8,7 +8,7 @@ export default class Read extends Command {
     help: flags.help({char: 'h'}),
   }
 
-  static args = [{name: 'channel'}]
+  static args = [{name: 'channel', required: true}]
```

Comment:
Not summer familiar with `@oclif/command` and could not find that option in any docs, but I assume it's there :)

Reply:
yeah I tested it- Surprised it's not documented